source/ch06b_PrepLinuxRobot.rst

```diff
@@ -143,6 +179,14 @@ Enter the IP Address or Name of the Linux Robot Controller into Phoenix tuner.
 .. |_Linux_IP_Image_| image:: img/LinuxWlan.png
 
 
+Your program runs the diagnostics server, so you should not install the diagnostics server through Phoenix Tuner.
```

Comment:
Start with a new section here that describes Diagnostic Server. Make sure it explains there are two versions, standalone and application, and how the application version is the ideal version to use.